code/_onclick/click.dm

```diff
@@ -82,18 +82,22 @@
 		return
 
 	var/obj/item/W = get_active_hand()
+	var/item_attack_delay = 0
+
+	if(W)
```

Comment:
Why would this check be necessary instead of just having the next line after if(W == A)